app/models/manager_refresh/inventory/core.rb

```diff
@@ -0,0 +1,41 @@
+module ManagerRefresh::Inventory::Core
```

Comment:
Right, this will get huge soon, so that is why I was started to divide it by managers